src/gc/gcimpl.h

```diff
@@ -157,8 +157,8 @@ class GCHeap : public IGCHeapInternal
     //returns the generation number of an object (not valid during relocation)
     unsigned WhichGeneration (Object* object);
     // returns TRUE is the object is ephemeral 
-    BOOL    IsEphemeral (Object* object);
-    BOOL    IsHeapPointer (void* object, BOOL small_heap_only = FALSE);
+    bool IsEphemeral (Object* object);
+    bool IsHeapPointer (void* object, bool small_heap_only = FALSE);
```

Comment:
>FALSE [](start = 61, length = 5)

any reason why you are not changing FALSE to false here? same with GetValidSegmentSize.